jax/lax/lax.py

```diff
@@ -1126,7 +1126,7 @@ def _safe_mul(x, y): return safe_mul_p.bind(x, y)
 ### convenience wrappers around traceables
 
 
-def conv(lhs, rhs, window_strides, padding, precision=None):
+def conv(lhs, rhs, window_strides, padding, precision=None, convolve=False):
```

Comment:
Can we avoid the modification to `lax.py`? Perhaps you can just call `lax.conv_general_dilated` directly from the `np.convolve` implementation, and basically move these lines over there.